src/graph.impl.cc

```diff
@@ -458,8 +460,35 @@ namespace hpp {
         }
       }
 
+      void Graph::getNumericalConstraints(const Long graphComponentId, hpp::Names_t_out names)
+	throw(hpp::Error)
+      {
+	if (!graph_)
+	  throw Error("You should create a graph");
+	graph::GraphComponentPtr_t elmt = graph::GraphComponent::get(graphComponentId).lock();
+	if (!elmt)
+	  throw Error("The component doesn't exists");
+	core::NumericalConstraints_t constraints = elmt->numericalConstraints();
+	core::LockedJoints_t lockedJoints = elmt->lockedJoints();
```

Comment:
`Graph::getNumericalConstraints` shouldn't returns the `LockedJoint`s. Please, add `Graph::getLockedJoints`.